dev/com.ibm.ws.webcontainer.security.app/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -86,3 +89,9 @@ includePathInWASReqURL.desc=Setting the Path parameter can allow the client/brow
 
 trackLoggedOutSSOCookies=Track logged out LTPA single signon tokens
 trackLoggedOutSSOCookies.desc=Specifies whether to track LTPA single signon tokens that are logged out on a server so that it can not be reused on the same server.
+
+overrideHttpAuthenticationMechanism=Name of the container provided HttpAuthenticationMechanism to override the application configured HttpAuthenticationMechanism. The HttpAutheMechanism is defined by the Java EE 8 Security specification.
+overrideHttpAuthenticationMechanism.desc=Specifies HttpAuthenticationMechanism name for overriding an application defined HttpAuthenticationMechanism. The acceptable value is BASIC or FORM. When FORM is used, loginFormURL and loginErrorURL need to be specified.
```

Comment:
Suggested changes:
This mechanism specifies the HttpAuthenticationMechanism name for overriding an application-defined HttpAuthenticationMechanism. The acceptable value is BASIC or FORM. When FORM is used, specify the loginFormURL and loginErrorURL URLs.

Reply:
How about this?
Specifies the HttpAuthenticationMechanism for overriding an application-defined HttpAuthenticationMechanism. The HttpAutheMechanism is defined by the Java EE 8 Security specification. The acceptable value is BASIC or FORM. When FORM is used, specify the loginFormURL and loginErrorURL properties.